plankton/adapters/epics.py

```diff
@@ -23,10 +23,16 @@
 from datetime import datetime
 
 from . import Adapter, ForwardProperty
-from pcaspy import Driver, SimpleServer
 from six import iteritems
 
-from plankton.core.utils import seconds_since
+from plankton.core.utils import seconds_since, From
+from plankton.core.exceptions import PlanktonException, StubAccessException
+
+# pcaspy might not be available. To make EPICS-based adapters show up
+# in the listed adapters anyway dummy types are created in this case
+# and the failure is postponed to runtime, where a more appropriate
+# PlanktonException can be raised.
+Driver, SimpleServer = From('pcaspy').import_or_stub('Driver', 'SimpleServer')
```

Comment:
What do you think about this approach? It's still delaying the ImportError, but it's completely general now and can be used wherever we have this kind of optional dependency. First I had a function `def import_or_stub(module, *names)`, but it looked weird. This is not overly pretty either, but it reads in a similar way as normal import statements.